React/prop-types.md

```diff
@@ -90,7 +86,7 @@ A `defaultProp` attribute like above ensures that every time the given props hav
 You can also create a custom validator which you can look up along with more examples of using `PropTypes` on the [PropTypes github page](https://github.com/facebook/prop-types).
 
 ## Key Takeaway
-`propTypes` is a static attribute you can define in a React component class which lists the props the component can take along with their types.  Mismatches or missing props will generate error messages in the browser console.  PropTypes help you document the props a component can take and help prevent the wrong types of props being passed accidentally.
+`propTypes` is an attribute you can define in a React component which lists the props the component can take along with their types.  Mismatches or missing props will generate error messages in the browser console.  PropTypes help you document the props a component can take and help prevent the wrong types of props being passed accidentally.
```

Comment:
So... students won't know what `static` is in terms of JavaScript officially by this point? We have definitions for it in Ruby but I think we still call them class methods and never static, and we have a definition of `static` methods for JavaScript classes on Wednesday (this lesson is slated for Tuesday)... Also, a static method on a class isn't quite the same as what's happening here...

Not sure what to do except maybe have a one sentence explanation somewhere.

Reply:
Aside from just being static, it's weird b/c we're attaching a property to a function, which they haven't seen before.

Reply:
"functions are objects and have properties"?